### ISSUE TITLE
Serialize annotation property values which are references as objects

### ISSUE THREAD

**Issue by ledsoft** [kbss-cvut/jb4jsonld#13]
As a user, I want JB4JSON-LD to serialize annotation property values as objects in case they represent individual references.

That is, if the annotation property value is a URI/URL, it should be serialized as an object with a single attribute - `@id`.

**Comment by ledsoft** [comment]
Example:

```Java
class A {

@OWLAnnotationProperty(iri = "http://example.org/annotation")
private Object annotation;
}

A a = new A();
a.annotation = URI.create("http://example.org");
```
Should be serialized as (ID omitted):
```JSON
{
    "http://example.org/annotation": {
        "@id": "http://example.org"
    }
}
```

**Comment by ledsoft** [comment]
Implemented in 0.5.0.
